samples/boards/microbit/pong/README.rst

```diff
@@ -0,0 +1,17 @@
+.. _microbit_pong:
+
+BBC micro:bit pong game
+#######################
+
+Overview
+********
+
+Play pong over Bluetooth between two micro:bit devices.
+
+Building
+********
+
+.. code-block:: console
+
+   $ cd samples/boards/microbit/pong
+   $ make
```

Comment:
Need a "Running" section to explain how to play the game.  How does it start and end?  Is a score shown?